webapp/public/javascripts/controllers.js

```diff
@@ -60,4 +60,67 @@ lunchrControllers.controller('RegisterController', ['$scope', '$http', '$state',
                     $scope.errorMessages = data;
                 })
         }
+    }]);
+
+lunchrControllers.controller('MapController', ['$scope',
+    function ($scope) {
+
+        $scope.map = { center: { latitude: 49.8651559, longitude: -97.11077669999997 }, zoom: 14 };
```

Comment:
Those are just some coordinates I pre-looked up for something else. It's some place on st Mary's. We can just use the coords for Winnipeg if someone finds them